_partials/integrations/hibernate-ogm.html.haml

```diff
@@ -0,0 +1,3 @@
+The Hibernate team maintains the
```

Comment:
@emmanuelbernard I felt the need to clarify that, as this will appear on the page were normally we'd host additional downloads from Infinispan.

Please see the preview: I don't think we'll want to update the release table for each OGM release.